### DISCUSSION TITLE
Add and customise title page when converting from MD to ODT

### DISCUSSION THREAD

**Discussion by jchwenger** [jgm/pandoc#9551] [Q&A]
Hi everyone,

I've been digging into customizing my markdown to odt pipeline, adding pagebreaks with lua filters, using `metadata.yaml` as a separate file for title and author, and such niceties.

Yet so far one thing escapes me. I have modified the `reference.odt` file in `~/.local/share/pandoc` to have, for instance, a page break after "author" (there might be better ways to create a title page, e.g. add a page break in the yaml file instead, still looking into this), and in `reference.odt` I changed that first page, where I have "TItlle" and "author", to have the First Page format, and had the page number begin at 1 after it. I save, but unlike other changes I made, this has no effect.

Is it because the changes made (e.g. adding a page break to the 'Author' style) are saved to `reference.odt` as a *document*, and not to its style? Is there a way to automate the creation of a title page without page numbers?

I should say I'm on MacOS Sonoma 14.0 with pandoc 3.1.8 from Homebrew.

Thanks a lot in advance!

**Comment by jgm** [comment]
> Is it because the changes made (e.g. adding a page break to the 'Author' style) are saved to reference.odt as a document, and not to its style?

Exactly. Only style changes matter. Is it possible to adjust the author *style* to add a page break after?

Have you played around with modifying the `opendocument` template?  This is used by the ODT writer, because the content is in opendocument format.

**Reply by jchwenger** [reply]
Oh, sorry, I might not have been too clear: playing around with the `opendocument` template works like a charm, and I am able to add a page break after a style (like author).

My question now is: is it possible *also* to have an automatically generated first page format for the title page, without page numbers, and the page numbers starting after? I can save the `opendocument` template with this, but that has no effect... I suppose that might stretch the limits of conversion?
